lib/validators/RequestValidator.js

```diff
@@ -67,6 +67,9 @@ class RequestValidator {
         return value !== 'application/offset+octet-stream';
     }
 
+    static _invalidAuthorizationHeader() {
```

Comment:
I don't see this used anywhere, is it leftover and no longer needed?

Reply:
I believe it pertains to #99, which we should probably address in a separate pull request.

Reply:
yes, you're right. it belongs in a separate PR. i will fix that and create another PR asap.

Reply:
fyi: https://github.com/tus/tus-node-server/pull/103